package OOP.Concurrency;

public class Main2 {
//Arithmetic exeptions
    public static void main(String[] args) {

        int a= 2;
        int b = 0 ;
        String name = null;

        try{
            name.equals("Sonic");
        }
        catch (Exception e) {
            //Used for debugging
            e.printStackTrace();

            System.out.println("Name was null");
        }


        }
}
